Breadcrumbs (fix #3218) (#3256)
canonical-web-and-design/vanilla-framework: templates/docs/component-status.md

    </tr>
  </thead>
  <tbody>
    <!-- 2.18 -->
    <tr>
      <th><a href="/docs/base/forms#checkbox">Breadcrumbs</a></th>
      <td><div class="p-label--updated">Updated</div></td>
      <td>2.18.0</td>

>       <td>We updaed the markup of the `breadcrumbs` component to comply with accessibility requirements. The class `.p-breadcrumbs` is now moved onto a `nav` element, the unordered list has been changed to an ordered one that has a class `.p-breadcrumbs__items`. </td>

Reviewer: You need to use HTML `<code>` in here, not markdown

Reviewer: ```suggestion
      <td>We updated the markup of the breadcrumbs component to comply with accessibility requirements. The class <code>.p-breadcrumbs</code> is now moved onto a <code>&lt;nav&gt;</code> element, the unordered list has been changed to an ordered one that has a class <code>.p-breadcrumbs__items</code>. </td>
```

    </tr>
  </thead>
  <tbody>
    <!-- 2.18 -->
    <tr>

>       <th><a href="/docs/base/forms#checkbox">Breadcrumbs</a></th>

Reviewer: Just noticed...
```suggestion
      <th><a href="/docs/patterns/breadcrumbs">Breadcrumbs</a></th>
```

      <td><div class="p-label--updated">Updated</div></td>
      <td>2.18.0</td>
      <td>We updated the markup of the breadcrumbs component to comply with accessibility requirements. The class <code>.p-breadcrumbs</code> is now moved onto a <code>&lt;nav&gt;</code> element, the unordered list has been changed to an ordered one that has a class <code>.p-breadcrumbs__items</code>. </td>
    </tr>
    <tr>

>       <th><a href="/docs/base/forms#checkbox">Breadcrumbs</a></th>

Reviewer: ```suggestion
      <th><a href="/docs/patterns/breadcrumbs#deprecated-markup">Breadcrumbs</a></th>
```

>       <td><div class="p-label--updated">Deprecated</div></td>

Reviewer: Wrong label type:

```suggestion
      <td><div class="p-label--deprecated">Deprecated</div></td>
```
